hotelReservation/helm-chart/hotelreservation/charts/wrk2/templates/deployment.yaml

```diff
@@ -0,0 +1 @@
+{{ include "hotelreservation.templates.wrk2-deployment" . }}
```

Comment:
Previously, this deployment was coming directly from `hotelreservation/templates/wrk2_deployment.tpl` but that didn't give it access to local `values.yaml` content.  By adding a `define` operator at the top of that file, it's now behaving like the other templates in the project and being deployed with its own environment.